---
int32 predicted_class_index